docs/static/releasenotes.asciidoc

```diff
@@ -15,6 +15,31 @@ This section summarizes the changes in the following releases:
 [[logstash-8-1-0]]
 === Logstash 8.1.0 Release Notes
 
+[[known-issue-8-1-0]]
+==== Known issue
+
+Uninstalling a plugin using `bin/logtash-plugin uninstall` may
+result in an error:
+
+```
+Gem::LoadError: You have already activated jruby-openssl 0.12.2, but your Gemfile requires jruby-openssl 0.12.1. Prepending `bundle exec` to your command may solve this.
+```
+
+Logstash should still run, and other plugin operations, such as `update` and `install`, should work as expected.
+
+NOTE: The `bin/logstash-plugin list` command may fail with the same error after a failed uninstallation.
+
+**Resolution**
+
+A successful plugin `update` will resolve this issue, and allow subsequent `uninstall` and `list` operations to
+work without issue.
+
+To update a plugin, run `bin/logstash-plugin update logstash-<plugintype>-<plugin>` where `logstash-<plugintype>-<plugin>` is the name of the plugin. 
+For example:`bin/logstash-plugin update logstash-filter-dissect`. 
```

Comment:
My thinking of being explicit with `logstash-filter-dissect` is that a plugin update needs to occur - ie the action of `bin/logstash-plugin update` has to actually update a plugin, because a newer version of a plugin is available. There are only a limited number of these plugins.

Reply:
Thanks for the explanation. Good info.